vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -58,7 +58,8 @@ const (
 	scaleCacheEntryFreshnessTime time.Duration = 10 * time.Minute
 	scaleCacheEntryJitterFactor  float64       = 1.
 	defaultResyncPeriod          time.Duration = 10 * time.Minute
-	defaultRecommenderName                     = "default"
+	// DefaultRecommenderName denotes the current recommender name as the "default" one.
```

Comment:
This comment seems incorrect. How about

// If recommender name is equal to DefaultRecommenderName it will handle VPA object which don't specify recommender name explicitly (and so implicitly specify that the default recommender should handle them)